apps/paymaster-proxy/src/initializeApiServer.ts

```diff
@@ -20,5 +24,20 @@ export const initializeApiServer = async (
     res.json({ ok: true })
   })
 
+  app.use(express.json(), ((err, req, res, next) => {
```

Comment:
middleware to catch req.body parsing errors and return

Reply:
would turn this into a function with a name so it's more clear what this is doing, would even recommend creating a middleware directory and just throw them all in there